src/test/java/co/org/osso/api/ApiApplicationTests.java

```diff
@@ -1,16 +1,40 @@
 package co.org.osso.api;
 
+import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
 
+import java.util.List;
+
 @RunWith(SpringRunner.class)
 @SpringBootTest
 public class ApiApplicationTests {
 
+	@Autowired
+	private BookController controller_target;
+
+	@Autowired
+	private CustomerService service_target;
+
+	@Test
+	public void checkBookServiceGetCustomers(){
+		List<Customer> customers = service_target.getCustomers();
```

Comment:
This looks weirdly formatted. Let's be consistent with tabs and spaces. Choose one or the other, and stick to it throughout the project.

Reply:
Okay. I'll be more attentive to it. Thanks.